python/BuiltInExample1.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML Image Data Reader'
+builtInExamplevti = XMLImageDataReader(FileName=['BuiltInExample1.vti'])
+builtInExamplevti.PointArrayStatus = ['Vectors_']
+
+# create a new 'Transform'
+transform1 = Transform(Input=builtInExamplevti)
+transform1.Transform = 'Transform'
+
+# init the 'Transform' selected for 'Transform'
+transform1.Transform.Rotate = [0.0, 0.0, -90.0]
+
+# create a new 'Compute Derivatives'
+computeDerivatives1 = ComputeDerivatives(Input=transform1)
+computeDerivatives1.Scalars = [None, '']
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.